appveyor.yml

```diff
@@ -1,17 +1,67 @@
-os: Visual Studio 2022
+image:
+  - Ubuntu2204
+  - Visual Studio 2022
 
-before_build:
-  - nuget restore Renci.SshNet.sln
+services:
+  - docker
 
-install:
-  - cinst dotnet-sdk --version=7.0.403 --limit-output
 
-build:
-  project: Renci.SshNet.sln
-  verbosity: minimal
 
-test_script:
-- cmd: >-
-    vstest.console /logger:Appveyor test\Renci.SshNet.Tests\bin\Debug\net462\Renci.SshNet.Tests.dll /TestCaseFilter:"TestCategory!=integration" --blame
+for:
+-
+  matrix:
+    only:
+      - image: Ubuntu2204
+  
+  environment:
+    APPVEYOR_BAKE_IMAGE: Ubuntu2204_dotnet_7_0_403  
```

Comment:
Do we have to change the image to `Ubuntu2204_dotnet_7_0_403` to make this work?

Reply:
It should work faster

Reply:
But do we need to set

```
image:
  - Ubuntu2204_dotnet_7_0_403
```

?

Reply:
I've tried this we do not need this

Reply:
Ok, cool. I was wondering because in https://www.appveyor.com/docs/custom-build-images/ it says

> To use that new image with SQL Server 2019 pre-installed in other projects specify its name in appveyor.yml:
> `image: win-sqlsrv-2019`